src/validations/css/prettycss.js

```diff
@@ -5,9 +5,34 @@ import endsWith from 'lodash/endsWith';
 
 const RADIAL_GRADIENT_EXPR =
   /^(?:(?:-(?:ms|moz|o|webkit)-)?radial-gradient|-webkit-gradient)/;
+
+const FILTER_VALUE_EXPR =
+  new RegExp([
+    'blur\\(',
+    'brightness\\(',
+    'contrast\\(',
+    'drop-shadow\\(',
+    'grayscale\\(',
+    'hue-rotate\\(',
+    'invert\\(',
+    'opacity\\(',
+    'saturate\\(',
+    'sepia\\(',
+    'inherit',
+    'initial',
+    'unset'].join('|'));
```

Comment:
@jwang1919 I am sorry I didn’t notice this before! But I think this expression is a bit too permissive, because it doesn’t have an anchor. For instance, it will allow something like:

```javascript
filter: verysillysepia(
```

Can we put a `^` anchor at the beginning?